static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -116,6 +117,7 @@ export function EventDetailsContent({
   const isANR = mechanism === 'ANR' || mechanism === 'AppExitInfo';
   const showPossibleSolutionsHigher = shouldShowCustomErrorResourceConfig(group, project);
   const groupingCurrentLevel = group?.metadata?.current_level;
+  const hasFeatureFlagSection = organization.features.includes('feature-flag-ui');
```

Comment:
How about:
```suggestion
  const hasFeatureFlagSection = organization.features.includes('feature-flag-issue-detail-ui');
```

Assuming we'll have other UI for FF that will have their own flag to control

Reply:
i was thinking we could gate everything flag-UI-related under the same flag -- it might be too much to handle flags for each spot🤔